### PR TITLE
Update to README.md

### PR DESCRIPTION
**Pull request by SB2020-eye**
...I'm trying to figure out how to do a pull request. Sorry if I got it wrong!

I thought I had already left my comment on my suggested edits. Just in case you don't see it, I think it's [here](https://github.com/SB2020-eye/ocrd_all/pull/1#issue-621229076).

(Moreover, I had an unfinished draft of the same comment keep showing up, and I couldn't figure out how to delete it. So if you see a comment that doesn't begin with "Postscript," please ignore it. Again, my apologies for any confusion.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -352,6 +354,17 @@ enabled by explicitly setting `OCRD_MODULES` or `DISABLED_MODULES`:
 * ocrd_kraken (currently unmaintained)
 * clstm (required only for ocrd_kraken)
 
+### Uninstall
+
+If you have installed ocrd_all natively and wish to uninstall, first remove the `ocrd_all` directory:
```
Comment (kba) [inline README.md:359]:
```suggestion
If you have installed ocrd_all natively and wish to uninstall, first `deactivate` the virtual environment  and remove the `ocrd_all` directory:
```

```
deactivate
rm -rf ocrd_all/venv
```

But more importantly, to uninstall, you don't need to delete the whole `ocrd_all` folder, just deactivate the venv and remove it -> no more `ocrd-*` commands.

File: README.md [hunk 2]
```diff
@@ -352,6 +354,17 @@ enabled by explicitly setting `OCRD_MODULES` or `DISABLED_MODULES`:
...
+```
+rm -rf ocrd_all
+```
+
+Next, remove all contents under ~/.parallel/semaphores:
+```
+rm -rf /~/.parallel/semaphores/*
```
Comment (kba) [inline README.md:366]:
```suggestion
rm -rf ~/.parallel/semaphores
```

There should be no remaining semaphores though, so if there are, something went wrong.

**Review by kba — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SB2020-eye) [reply README.md:359]: 👍

File: README.md (hunk 2 above)
Comment (SB2020-eye) [reply README.md:366]: 👍

**Comment by SB2020-eye** [comment]
Thanks, @kba. I'm not sure if it's required for me to do anything else on my end or not. Please let me know if there is. I'm definitely fine with whatever you decide should be included or altered, or else not included at all. Take care

**Review by kba — APPROVED** [review]

**Pull request merged**